README.md

```diff
@@ -329,6 +329,32 @@ if (hasNextPage) {
 }
 ```
 
+#### Resume pagination
```

Comment:
@DebbieAtSeam @phpnode I considered extending the SDK with something that saved and  consumed the nextPageUrl, but that turned out to be pretty complicated in practice. I think this is simple enough and more transparent anyway.